### PR TITLE
RolesAnywhere-3788: Include token TTL header in token API response

### PR DESCRIPTION
**Pull request by 13ajay**
*Description of changes:*

Include token TTL header in the response to the token API when serving a local endpoint.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws_signing_helper/serve.go [hunk 1]
```diff
@@ -158,6 +158,7 @@ func AllIssuesHandlers(cred *RefreshableCred, roleName string, opts *Credentials
 		expirationTime := time.Now().Add(time.Second * time.Duration(tokenTTL))
 		InsertToken(token, expirationTime)
 
+        w.Header().Set(EC2_METADATA_TOKEN_HEADER, tokenTTLStr)
```
Comment (rlalcorn) [inline aws_signing_helper/serve.go:161]: Stay consistent w/ whitespace.

**Review by rlalcorn — CHANGES_REQUESTED** [review]

File: aws_signing_helper/serve.go (hunk 1 above)
Comment (13ajay) [reply aws_signing_helper/serve.go]: Thanks for the catch - I've fixed it in my next revision.

File: aws_signing_helper/serve.go [hunk 2]
```diff
@@ -228,6 +270,9 @@ func Serve(port int, credentialsOptions CredentialsOpts) {
 	refreshableCred.SecretAccessKey = credentialProcessOutput.SecretAccessKey
 	refreshableCred.Token = credentialProcessOutput.SessionToken
 	refreshableCred.Expiration, _ = time.Parse(time.RFC3339, credentialProcessOutput.Expiration)
+	refreshableCred.Code = "Success"
```
Comment (rlalcorn) [inline aws_signing_helper/serve.go:273]: Extract "Success" and "AWS-HMAC" to constants, maybe?

Comment (13ajay) [reply aws_signing_helper/serve.go]: I've changed this and added another commit to this PR.

**Review by rlalcorn — APPROVED** [review]

**Pull request merged**